{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "glusterfs",
                    "vendor": "unspecified",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "glusterfs 3.10.12"
                        },
                        {
                            "status": "affected",
                            "version": " glusterfs 4.0.2"
                        }
                    ]
                }
            ],
            "datePublic": "2018-04-19T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "glusterfs server before versions 3.10.12, 4.0.2 is vulnerable when using 'auth.allow' option which allows any unauthenticated gluster client to connect from any network to mount gluster storage volumes. NOTE: this vulnerability exists because of a CVE-2018-1088 regression."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "attackComplexity": "LOW",
                        "attackVector": "ADJACENT_NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 8.0,
                        "baseSeverity": "HIGH",
                        "confidentialityImpact": "HIGH",
                        "integrityImpact": "HIGH",
                        "privilegesRequired": "NONE",
                        "scope": "UNCHANGED",
                        "userInteraction": "REQUIRED",
                        "vectorString": "CVSS:3.0/AV:A/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
                        "version": "3.0"
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-287",
                            "description": "CWE-287",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2020-01-20T06:06:12",
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat"
            },
            "references": [
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2018-1112"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://review.gluster.org/#/c/19899/1..2"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://access.redhat.com/articles/3422521"
                },
                {
                    "name": "RHSA-2018:1268",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2018:1268"
                },
                {
                    "name": "RHSA-2018:1269",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_REDHAT"
                    ],
                    "url": "https://access.redhat.com/errata/RHSA-2018:1269"
                },
                {
                    "name": "openSUSE-SU-2020:0079",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_SUSE"
                    ],
                    "url": "http://lists.opensuse.org/opensuse-security-announce/2020-01/msg00035.html"
                }
            ]
        }
    },
    "cveMetadata": {
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "assignerShortName": "redhat",
        "cveId": "CVE-2018-1112",
        "datePublished": "2018-04-25T12:00:00",
        "dateReserved": "2017-12-04T00:00:00",
        "dateUpdated": "2020-01-20T06:06:12",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0"
}
